apps/snitch_core/lib/core/data/schema/order.ex

```diff
@@ -93,7 +96,14 @@ defmodule Snitch.Data.Schema.Order do
       |> get_field(:line_items)
       |> LineItemModel.compute_total()
 
-    total = Enum.reduce([item_total], &Money.add!/2)
+    packages_total =
+      if order_changeset.data.id != nil do
```

Comment:
Reason to check `order_id` to `nil`, there are few test cases, where order is generated with out `id`.
I am not sure how its possible

Reply:
We don't want to manage packages from an order.

Reply:
The order `:id` is set only when the struct is inserted into the DB. Many test cases don't touch the repo.